indico/modules/events/editing/client/js/store.js

```diff
@@ -0,0 +1,12 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import createReduxStore from 'indico/utils/redux';
+import reducer, {initialState} from './reducer';
+
+export default staticData =>
+  createReduxStore('editing-timeline', {timeline: reducer}, {timeline: initialState, staticData});
```

Comment:
do we need the `timeline: initialState` here? it's already provided in the reducer's default argument